elide-spring/elide-spring-boot-autoconfigure/src/test/java/com/yahoo/elide/spring/App.java

```diff
@@ -15,6 +16,7 @@
  */
 @Slf4j
 @SpringBootApplication
+@EntityScan(basePackages = {"${elide.modelPackage}"})
```

Comment:
I don't think we need this.  Why did we add?

Reply:
If we don't this, the Async Models are bound by default to API and swagger. We wanted to avoid that and are explicitly telling it to bind/discover the entities in the provided package.

Reply:
@aklish Since we are putting `@EntityScan(basePackageClasses = AsyncQuery.class)` in the `ElideAsyncConfiguration.java`, it overrides the default behavior of spring to scan all the entities in the package where App.java (with annotation @SpringBootApplication) will be placed. In my testing, I found that we can change `@EntityScan(basePackages = {"${elide.modelPackage}"})` to `@EntityScan` in this class and it should work fine. This will be a simple change for users that can be included in the documentation. 
The other option would be as suggested by you that we include the logic from the dynamic config design to use our own implementation of `LocalContainerEntityManagerFactoryBean` and add the Async models ourselves but it could potentially take time to implement and test out fullt. Let me know your thoughts.